.gitignore

```
@@ -0,0 +1,106 @@
+# Logs
```

```suggestion
*~
*[._]sw[g-p]
.eslintcache

node_modules/
.idea/
.vscode/
```

I agree with @jeddy3.
KISS